### ISSUE TITLE
Is it hard, or would it be possible to support youtube shorts urls?

### ISSUE THREAD

**Issue by aaronkchsu** [cookpete/react-player#1427]
I'm curious if there are plans or if anyone thinking would be possible to get react-player to work with youtube shorts? 

https://www.youtube.com/shorts/7dnri1wKlH4

**Comment by amakk** [comment]
I would also like to see this implemented. For now I got it to work by modifying the video url as follows (at least for the URLs I tested). So seems like it would be a pretty simple change to the code. 

```
if (url?.includes?.('youtube.com/shorts')) {
  return url.split('?')[0].replace('shorts/', 'watch?v=');
}
```

**Comment by Paul-Burinda** [comment]
I would also like to see this implemented.  Thanks @amakk for the current workaround until then.

**Comment by cookpete** [comment]
Fixed in `2.10.1`
